src/index.html

```diff
@@ -7,15 +7,12 @@
     <link rel="stylesheet" href="https://js.arcgis.com/4.10/esri/css/main.css">
     <link rel="stylesheet" href="index.css">
     <script>
-      // Current path minus everything after the last forward slash
-      var locationPath = location.pathname.replace(/\/[^\/]+$/, "");
-      // Dojo wants a very explicit path in order to load our JS package
       window.dojoConfig = {
         isDebug: true,
         packages: [
           {
             name: "app",
-            location: location.protocol + "//" + location.host + locationPath + "/app"
+            location: location.origin + "/app"
```

Comment:
Just realized that this is bad. Won't work if this app is deployed to e.g. a subdirectory.